journal/src/main/java/io/zeebe/journal/JournalReader.java

```diff
@@ -90,4 +90,7 @@
    * @return the index of the record that will be returned by {@link #next()}
    */
   long seekToAsqn(long asqn);
+
+  @Override
+  void close();
```

Comment:
Why is this needed? The interface extends `AutoCloseable`.

Reply:
To avoid handling the exception - none of the consumers expect close to throw an exception. I could avoid this and make it bubble up, or we can handle the error. I would prefer the latter, but I would also propose to do it as a follow up.

Reply:
:+1: